CHANGELOG.rst

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+0.6.0 (unreleased)
```

Comment:
Will we release?

Reply:
There's no rush, the old wheels still work on Python 3.9+.